src/probes/journal.c

```diff
@@ -224,10 +224,14 @@ static bool add_filters(sd_journal *journal)
 
         MATCH(data);
         free(data);
+        AND;
         // The three highest log levels, all indicating errors
         MATCH("PRIORITY=1");
         MATCH("PRIORITY=2");
         MATCH("PRIORITY=3");
+        OR;
```

Comment:
The journal probe is in the same boat as the crash probe... No unit tests are written for either because they rely on system state for their operation, and I'm not sure how I would go about mocking everything for the tests.